src/components/JobDetailCard.jsx

```diff
@@ -0,0 +1,80 @@
+import styled from '@emotion/styled';
+
+// Temporarily importing JSON here to test code
+import data from '../assets/data.json';
+
+const JobDetailContainer = styled.div`
+  padding: ${props => props.theme.layout.space500};
+  margin: 0 auto;
+  border-radius: 6px;
+  background-color: ${props => props.theme.colors.white};
+  max-width: 730px;
+`;
+
+const JobDetailHeader = styled.div`
+  color: ${props => props.theme.colors.darkGray};
+  display: flex;
+  align-items: center;  
+  justify-content: space-between;
+`;
+
+const JobTitle = styled.h1`
+  color: ${props => props.theme.colors.darkBlue};
+  margin-top: ${props => props.theme.layout.space50};
+  margin-bottom: ${props => props.theme.layout.space100};
+`;
+
+const JobLocation = styled.div`
+  font-size: ${props => props.theme.typography.fs87};
+  font-weight: ${props => props.theme.typography.fwBold};
+  color: ${props => props.theme.colors.violet};
+`;
+
+const JobDetailDescription = styled.div`
+  p, ul, ol {
+    color: ${props => props.theme.colors.darkGray};
+  }
+`;
+
+export const JobDetailCard = () => {
```

Comment:
Let's use `export default` for our components since we want the component to be the main export of each module. This helps maintain consistency across our codebase and aligns with common React practices. 

For more information about the difference between export default and named exports, see this article [here](https://medium.com/@heshramsis/understanding-the-difference-between-export-default-and-export-with-named-exports-in-javascript-f0569c221a3).

```suggestion
const JobDetailCard = () => {
```

Reply:
Ouu, cool, I always wondered what the differences are but never looked into it. Good to know!